debezium-core/src/main/java/io/debezium/transforms/ExtractNewRecordState.java

```diff
@@ -83,6 +100,8 @@ public void configure(final Map<String, ?> configs) {
         delegateConfig.put("static.field", ExtractNewRecordStateConfigDefinition.DELETED_FIELD);
         delegateConfig.put("static.value", "false");
         updatedDelegate.configure(delegateConfig);
+
+        schemaUpdateCache = new SynchronizedCache<>(new LRUCache<Schema, Schema>(16));
```

Comment:
@adriank-convoy Please eaxtract the cahce size as a constant and increse it to 64. If needed we might make it configurable in future.